pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/MhloLegalizeUtils.cpp

```diff
@@ -20,21 +20,41 @@
 
 #include <mlir-hlo/Dialect/mhlo/IR/hlo_ops.h> // from tf repo
 
+#include <iostream>
 #include "torch-mlir/Conversion/TorchToMhlo/MhloLegalizeUtils.h"
 
 namespace mlir {
 namespace mhlo {
 
+std::vector<int64_t> rangeIndices(int64_t min, int64_t max) {
+  std::vector<int64_t> range;
+  for (int64_t k = min; k < max; ++k) {
+    range.push_back(k);
+  }
+  return range;
+}
+
+std::vector<int64_t> normalizeDimIndex(ArrayRef<int64_t> dims, int64_t rank) {
+  std::vector<int64_t> newDims;
+  newDims.reserve(rank);
+  std::transform(
+      dims.begin(),
+      dims.end(),
+      std::back_inserter(newDims),
+      [rank](int64_t d) -> int64_t { return (d + rank) % rank; });
+  return newDims;
+}
+
 // Create a 32-bit float constant operator from a float
 Value getMhloConstTensorSingleF32(
     PatternRewriter& rewriter,
     Operation* op,
     float val) {
-  auto const_type = RankedTensorType::get({}, rewriter.getF32Type());
-  auto const_attr = DenseElementsAttr::get(const_type, val);
+  auto constType = RankedTensorType::get({}, rewriter.getF32Type());
+  auto const_attr = DenseElementsAttr::get(constType, val);
```

Comment:
using `constAttr` as well?

Reply:
I'm completely confused by mixed this coding styling ...